react/react-addons-test-utils.d.ts

```diff
@@ -6,7 +6,7 @@
 /// <reference path="react.d.ts" />
 
 declare namespace __React {
-    interface SyntheticEventData {
+    interface SyntheticEventData extends Event {
```

Comment:
I don't think this is okay. There are required properties on `Event`, but the `Simulate` API does not require any properties.